CMakeLists.txt

```diff
@@ -52,18 +52,19 @@ set(CMAKE_INSTALL_NAME_DIR ${CMAKE_INSTALL_PREFIX}/lib)
 set(CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH true)
 
-if (T8CODE_USE_SYSTEM_SC)
```

Comment:
Changing the position of this section conflicts with the fix in https://github.com/cburstedde/p4est/pull/301.
t8code most probably links to another libsc commit than p4est does. So t8code should have precedence in this.

Reply:
I didn't know that you had already started fixing the p4est BS and this was supposed to prevent the double `SC::SC` target. I will revert this change